File: JavaScript/8js 2/ghost.js
/*
|--------------------------------------------------------------------------
| Color Ghost
|--------------------------------------------------------------------------
|
*/

function Ghost() {
   const colors = ["white", "yellow", "purple", "red"];
   this.color = colors[Math.floor(Math.random() * colors.length)];
}


class Ghost {
   constructor() {
      this.color = [`red`, `yellow`, `purple`, `white`][(Math.random() * 4) ^ 0];
   }
}


const Ghost = function () {
   this.color = ["white", "yellow", "purple", "red"][Math.floor(Math.random() * 4)];
};

File: Python/8py 2/ghost.py
"""
|--------------------------------------------------------------------------
| Color Ghost
|--------------------------------------------------------------------------
|
"""

from random import choice, randint, randrange
class Ghost:
   def __init__(self):
      colors = ['red', 'yellow', 'purple', 'white']
      self.color = choice(colors)


class Ghost(object):
   def __init__(self):
      self.color = ['white', 'yellow', 'purple', 'red'][randint(0, 3)]


class Ghost(object):
   def __init__ (self):
      self.color = ["white",'yellow','purple','red'][randrange (4)]